package/MDAnalysis/core/topologyattrs.py

```diff
@@ -880,6 +881,9 @@ def __init__(self, values):
         Eg:  [(0, 1), (1, 2), (2, 3)]
         """
         self.values = values
+        if types is None:
```

Comment:
I don't really like adding an array of Nones here, but I can see why you did it.  We'll fix this eventually